### PR TITLE
Refactor, change to Profiles instead of Services. Added json of dev, …

### PR DESCRIPTION
**Pull request by ayancovich**
…qa & prod instead of microservices.json (gateway,registry,auth,users)

### CODE REVIEW

File: .idea/libraries/config_1_0_0.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<component name="libraryTable">
```
Comment (nestorreloaded) [inline .idea/libraries/config_1_0_0.xml]: Please don't commit this files

File: .idea/libraries/lib.xml [hunk 2]
```diff
@@ -0,0 +1,147 @@
+<component name="libraryTable">
```
Comment (nestorreloaded) [inline .idea/libraries/lib.xml]: Same here

File: .idea/misc.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (nestorreloaded) [inline .idea/misc.xml]: Same here

File: .idea/misc.xml [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+<project version="4">
```
Comment (nestorreloaded) [inline .idea/misc.xml]: Same here

File: src/main/java/com/endava/config/ConfigConstants/ConfigConstants.java [hunk 5]
```diff
@@ -0,0 +1,15 @@
+package com.endava.config.ConfigConstants;
```
Comment (nestorreloaded) [inline src/main/java/com/endava/config/ConfigConstants/ConfigConstants.java:1]: Shouln't this configs be in the dababase too?

File: src/main/resources/application.properties [hunk 6]
```diff
@@ -19,10 +18,13 @@ eureka.client.fetch-registry=true
 #register url to server
 eureka.instance.hostname=${SERVER_URL}
 
+health.config.enabled=false
+#logging.level.org.springframework=DEBUG
```
Comment (nestorreloaded) [inline src/main/resources/application.properties]: Please remove the debug setting

**Review by nestorreloaded — CHANGES_REQUESTED** [review]
Please check the recommendations, remove the files from IntelliJ and add them to .the .gitignore file

File: .idea/libraries/config_1_0_0.xml (hunk 1 above)
Comment (ayancovich) [reply .idea/libraries/config_1_0_0.xml]: Fixed

File: .idea/libraries/lib.xml (hunk 2 above)
Comment (ayancovich) [reply .idea/libraries/lib.xml]: Fixed

File: .idea/misc.xml (hunk 3 above)
Comment (ayancovich) [reply .idea/misc.xml]: Fixed

File: .idea/misc.xml (hunk 4 above)
Comment (ayancovich) [reply .idea/misc.xml]: Fixed

File: src/main/java/com/endava/config/ConfigConstants/ConfigConstants.java (hunk 5 above)
Comment (ayancovich) [reply src/main/java/com/endava/config/ConfigConstants/ConfigConstants.java:1]: Fixed

File: src/main/resources/application.properties (hunk 6 above)
Comment (ayancovich) [reply src/main/resources/application.properties]: Fixed

**Review by nestorreloaded — APPROVED** [review]

File: src/main/main.iml [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/java" isTestSource="false" />
+    </content>
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="library" name="lib" level="project" />
+  </component>
+</module>
```
Comment (en-nbantar) [inline src/main/main.iml]: Is this file needed or it's auto-generated?

**Review by en-nbantar — CHANGES_REQUESTED** [review]

File: src/main/main.iml (hunk 7 above)
Comment (nestorreloaded) [reply src/main/main.iml]: It was generated by intelliJ, we should not use it

Comment (ayancovich) [reply src/main/main.iml]: File removed.

**Review by en-nbantar — APPROVED** [review]
